### PR TITLE
Replace Travis with GitHub actions

### PR DESCRIPTION
**Pull request by SergeAstapov**
Wonder if maintainer is up to this change.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-  - git config --global core.autocrlf true
-
-# Test against these versions of Node.js.
-environment:
-  matrix:
-    - nodejs_version: "8"
-    - nodejs_version: "10"
-image:
-  - Ubuntu
-  - Visual Studio 2017
```
Comment (SergeAstapov) [inline appveyor.yml:14]: I'll add Windows to testing matrix

**Comment by quaertym** [comment]
+1 from me. Let's remove appveyor as well in favor of github actions.

**Comment by SergeAstapov** [comment]
> +1 from me. Let's remove appveyor as well in favor of github actions.

@quaertym the second commit in this PR does it, see "[Remove AppVeyor config](https://github.com/quaertym/ember-cli-dependency-checker/pull/130/commits/489956a5561a5399141caad169a111dcafd773dd)"

**Comment by quaertym** [comment]
We can remove the appveyor badge from README and add windows to matrix as you said.

**Comment by SergeAstapov** [comment]
@quaertym done! tests results - https://github.com/SergeAstapov/ember-cli-dependency-checker/pull/2/checks

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: "Tests - ${{ matrix.os }} - Node.js ${{ matrix.node-version }}"
+    runs-on: "${{ matrix.os }}-latest"
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x]
```
Comment (SergeAstapov) [inline .github/workflows/ci.yml:21]: @quaertym note 12.x does not work and `yarn install` fails on Mac OS on fsevents installation.
I think all the dependencies needs to be updated to fix this.

Also, we may release a new major dropping support for older Node.js versions and bumping all the dependencies but it should be separate

Comment (quaertym) [reply .github/workflows/ci.yml:21]: Sounds good. Those that prefer can stay on the older version. But I want to release the bower changes before as a minor version first.

Comment (SergeAstapov) [reply .github/workflows/ci.yml:21]: @quaertym agree, that totally makes sense!

**Comment by quaertym** [comment]
Thanks, I merged this. 

I think we should follow ember-cli and use same node versions (12.x, 14.x, 16.x).  Any blockers for this?

**Comment by SergeAstapov** [comment]
Sorry for delay - yes, we had to bump dependencies to be able to use newer Node versions, ad it was done in https://github.com/quaertym/ember-cli-dependency-checker/pull/135

**Pull request merged**